File: safe_place/v1/get/safety_rated_locations_by_square/iOS SDK/SafePlace.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)


amadeus.safety.safetyRatedLocations.bySquare.get(params: ["north": "41.397158",
    "west": "2.160873",
    "south": "41.394582",
    "east": "2.177181"], onCompletion: {
        result in
        switch result {
        case .success(let response):
            print(response.data)
        case .failure(let error):
            print(error.localizedDescription)
        }
    })

File: safe_place/v1/get/safety_rated_locations_by_square/Node SDK/safe_place.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// How safe is Barcelona? (based on a square)
amadeus.safety.safetyRatedLocations.bySquare.get({
  north: 41.397158,
  west: 2.160873,
  south: 41.394582,
  east: 2.177181
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});